src/main/java/mate/academy/Main.java

```diff
@@ -6,15 +6,19 @@
 import mate.academy.service.AuthenticationServiceImpl;
 import mate.academy.service.OrderService;
 import mate.academy.service.OrderServiceImpl;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class Main {
+    private static final Logger logger = LogManager.getLogger(Main.class);
+
     public static void main(String[] args) {
         AuthenticationService authenticationService = new AuthenticationServiceImpl();
         User user;
         try {
             user = authenticationService.login("bob", "1234");
         } catch (AuthenticationException e) {
-            e.printStackTrace();
+            logger.error("Username or password are incorrect", e);
```

Comment:
```suggestion
            logger.warn("Username or password are incorrect", e);
```
тут краще робити warn замість error так як це не є помилокою роботи програми а просто опечатка від юзера, в еррор можна писати, коли якийсь сервіс не зміг коректно відпрацювати або дуже довго обробляв запит